helm/ingress/README.md

```diff
@@ -9,12 +9,13 @@ The following will create an nginx ingress controller and rules that route publi
 
 1. Deploy the ingress controller like so in the `monitoring` namespace:
 ```bash
-helm install stable/nginx-ingress --namespace monitoring --name grafana-ingress --set controller.replicaCount=2 --set rbac.create=true
-```
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+helm install grafana-ingress stable/nginx-ingress --namespace monitoring --name grafana-ingress --set controller.replicaCount=2 --set rbac.create=true
```

Comment:
I think we need to remove the `--name` arg as it seems to be no-longer used?

Reply:
yep, fixed. its treated as release name now up front